tests/tools/tools.spec.ts

```diff
@@ -57,6 +57,13 @@ describe('Tools', () => {
         const result = Tools.formatCurrencyValue(Number.MAX_SAFE_INTEGER, options);
         expect(result).eql('$ 9,007,199,254,740,991.00');
       });
+      it('should return formatted value without symbol', () => {
```

Comment:
It worth also testing what happened to a signed value:
- should format a signed value without a symbol